DeepCat.js

```diff
@@ -8,7 +8,8 @@
 ( function( $, mw ) {
 	var DeepCat = {},
 		keyString = 'deepcat:',
-		maxDepth = 15,
+		defaultDepth = 15,
+		maxDepth = 20,
```

Comment:
I agree, when we have a default in the DeepCat code and a "hard" limit in the jsonp interface we do not really another limit in the gadget. The hint text can then also be changed to "default depth" instead of "max" or something like that.